3Dmol/parsers.js

```diff
@@ -2202,5 +2202,52 @@ $3Dmol.Parsers = (function() {
 	}
         return atoms;
     }
+
+    /**
+     * Parse a lammps trajectory file from str and create atoms
+     */
+    parsers.lammpstrj = parsers.LAMMPSTRJ = function(str, options){
```

Comment:
In lammps, we dont have the concept of a topology file. All the information is present in the trajectory file itself.